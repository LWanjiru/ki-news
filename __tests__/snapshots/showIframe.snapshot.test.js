import React from 'react';
import renderer from 'react-test-renderer';
import ShowIframe from './../../src/scripts/components/showIframe'

describe('Show Iframe (Snapshot)', () => {
  it('renders an iframe', () => {
    const component = renderer.create(<ShowIframe />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
