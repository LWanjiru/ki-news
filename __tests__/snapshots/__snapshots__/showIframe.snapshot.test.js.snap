// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Show Iframe (Snapshot) renders an iframe 1`] = `
<div
  className="mt-3 embed-responsive embed-responsive-16by9 i-frame"
>
  <iframe
    className="border border-warning rounded"
    name="iframe_a"
  />
</div>
`;
